magicctapipe/resources/lstchain_standard_config_modified.json

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
why do you need the standard LST config file?? LST chain gets installed together with MCP, so you can take it from there

Reply:
We need the modified one to evaluate NSB for some particular runs (by adding the heuristic_flatfield lin, not present in the config installed in lstchain when installing MCP, some runs can be processed instead of producing a 'missing FF table' error)

Reply:
I can try to find out which are the needed entries to create the dictionary and modify it. But as far as I know I will have to produce a new file instead of passing the dictionary to the function (but I have to check lstchain)

Reply:
maybe fixed now (I load and modify the config in nsb_level and I save the (un)changed version to the current directory to use it in LSTnsb)